plugins/processors/multiplier/README.md

```diff
@@ -0,0 +1,30 @@
+# Multiplier Processor
+
+The multiplier processor is used to change the values of metrics with some multiplication factor.
+
+### Examples:
+
+* [input.system](./plugins/inputs/system) plugin produces metric [uptime](./plugins/inputs/system/SYSTEM_README.md#metrics) with amount of seconds after computer start. If it is needed to get value in hours multiplier configuration should contain
+"system uptime=0.00027777777"
+
+* Some **metric** contains **value** in range (0;&nbsp;1). If it is needed to get values in more human readable percentage format multiplier configuration should contain
```

Comment:
I assume that the range is supposed to be `[0; 1]`, unless 0 and 1 are indeed excluded. 🙂

Reply:
Thanks. Fixed.